docs/source/index.rst

```diff
@@ -54,52 +54,30 @@ easy.
 **Step 1:** Pick the magic string that will identify your library. To
 avoid collisions, this should match your library's name on PyPI.
```

Comment:
Lower case?  Case folded?  twisted is `Twisted`.  Doesn't super matter, just figured I'd comment in case you do feel like caring.

Reply:
```suggestion
avoid collisions, this should match your library's PEP 503 normalized name on PyPI.
```

?